src/zip.ts

```diff
@@ -28,6 +31,12 @@ const sumWrites = async (writes: Promise<number>[]): Promise<number> => {
   );
 };
 
+// const { createWriteStream } = streamSaver;
```

Comment:
whats this?

Reply:
Remnants of a snippet that was moved to API.ts. I've just deleted this